rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -697,7 +698,7 @@ int __pal_exec(char *path, char *argv[], pal_stdio_fds *stdio, int *exit_code)
 	}
 
 	fprintf(fp, "Enclave runtime skeleton initialization succeeded\n");
-	fclose(fp);
+	fflush(fp);
```

Comment:
why not fclose(fp)? There is one flush operation in fclose(fd)

Reply:
If fclose(fp), then in `__pal_destroy` function, `fdopen(pal_stdio.stderr)` will returns `NULL` with the `Bad file descriptor `